packages/mobile/src/fiatExchanges/FiatExchangeOptions.tsx

```diff
@@ -188,10 +176,7 @@ function FiatExchangeOptions({ route, navigation }: Props) {
               borderTopLeftRadius: 8,
               borderTopRightRadius: 8,
             }}
-            enabled={
-              (!SIMPLEX_DISABLED || selectedPaymentMethod !== PaymentMethod.FIAT) &&
-              selectedPaymentMethod !== PaymentMethod.ADDRESS
-            }
+            enabled={selectedPaymentMethod !== PaymentMethod.ADDRESS}
```

Comment:
`PaymentMethod.ADDRESS` means withdrawing funds to an address, we use the CELO withdrawal flow for that which doesn't support cUSD. Using this would be the same as just sending cUSD to someone.